diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LocalVariablesWarnTest.kt

```diff
@@ -549,4 +549,19 @@ class LocalVariablesWarnTest : LintTestBase(::LocalVariablesRule) {
                 """.trimMargin()
         )
     }
+
+    @Test
+    @Tag(WarningNames.LOCAL_VARIABLE_EARLY_DECLARATION)
+    fun `should not trigger on triple quoted strings`() {
+        lintMethod(
+                """
+                    |class Example {
+                    |    fun some() {
+                    |       val code = ${"\"\"\"  class Some {\n \t\t fun for() {} \n \t} \"\"\".trimIndent()"} 
```

Comment:
Why do you need `${}` here? Also, this is not exactly example from the issue, there are some different newlines there.

Reply:
It is used to insert triple quotes